### DISCUSSION TITLE
AvaloniaUI Web "An unhandled error has occurred"

### DISCUSSION THREAD

**Discussion by flesarradu** [AvaloniaUI/Avalonia#9536] [General]
Hi, I'm getting this error, I don't know why I added 2 simple buttons on the main page and the web app crashed after that. The desktop app works.
![image](https://user-images.githubusercontent.com/32610325/203801801-8daa8e1b-1cf0-46f0-bac1-157f1244527d.png)
Thank you!

**Comment by hez2010** [comment]
From the stack trace seems that you are trying to create a Window in the app, which is definitely not supported by the browser. 
You can check out the sample here: https://github.com/AvaloniaUI/Avalonia/tree/master/samples%2FControlCatalog.Browser.Blazor

**Reply by flesarradu** [reply]
Thank you very much! I'm new to this I thought it would manage it there. Thanks!
